math/lerp.go

```diff
@@ -0,0 +1,6 @@
+package math
+
+func Lerp(v0, v1, t float64) float64 {
```

Comment:
Please provide a good documentation comment for this function. Just having a wikipedia link is not a good strategy for contributing code to projects.